www/class/centreon-partition/partEngine.class.php

```diff
@@ -614,8 +614,8 @@ public function isCompatible($db)
             $dbResult->closeCursor();
 
             if (
-                stristr($dbType, "MySQL")
-                || stristr($dbType, "Source distribution")
+                (stristr($dbType, "MySQL")
+                || stristr($dbType, "Source distribution"))
```

Comment:
this makes condition more readable
```suggestion
                (
                    stristr($dbType, "MySQL")
                    || stristr($dbType, "Source distribution")
                )
```